spec/system/evidence_steps_helper.rb

```diff
@@ -1,100 +1,102 @@
-def and_i_add_a_new_piece_of_evidence(evidence_group, evidence_type)
-  @evidence_text = evidence_type
-  click_link 'Add evidence'
-  choose evidence_group
-  choose evidence_type
-  click_button 'Continue'
-end
-
-def and_i_answer_no_to_every_question
-  ['physical features', 'cryptographic security', 'authoritative source', 'cancelled'].each do |question_snippet|
-    expect(page).to have_content question_snippet
-    choose 'No'
+module EvidenceStepsHelper
+  def and_i_add_a_new_piece_of_evidence(evidence_group, evidence_type)
+    @evidence_text = evidence_type
+    click_link 'Add evidence'
+    choose evidence_group
+    choose evidence_type
     click_button 'Continue'
   end
-end
 
-def and_i_answer_yes_to_every_question
-  ['physical features', 'original', 'errors', 'document capture app', 'recognised guidance', 'taught how to tell'].each do |question_snippet|
-    expect(page).to have_content question_snippet
-    choose 'Yes'
-    click_button 'Continue'
+  def and_i_answer_no_to_every_question
+    ['physical features', 'cryptographic security', 'authoritative source', 'cancelled'].each do |question_snippet|
+      expect(page).to have_content question_snippet
+      choose 'No'
+      click_button 'Continue'
+    end
   end
 
-  expect(page).to have_content 'refresh their training'
-  choose 'Every year'
-  click_button 'Continue'
-
-  expect(page).to have_content 'official templates'
-  choose 'Yes'
-  click_button 'Continue'
+  def and_i_answer_yes_to_every_question
```

Comment:
This doesn't actually answer yes to every question, sounds somewhat misleading

Reply:
How about `and_i_answer_all_the_questions`?

Naming things is hard :-)

Reply:
This method only answers the first 6 questions before hitting a none 'yes' answerable question. So using the world `all` or `every` in `i_answer_yes_to_every_question` is what i find misleading. Naming is hard :(

Reply:
Yeah that sounds much better, or `and_i_answer_positively_to_all_questions`